RELEASING.md

```diff
@@ -0,0 +1,60 @@
+# Releasing Federation
+
+This is a quick and opinionated set of commands for building and releasing a new version of the packages in this monorepo.
+
+#### Start a release PR
+
+1. `FEDERATION_RELEASE_VERSION=X.Y.Z` (e.g. `2.0.1` or `2.0.0-beta.3` or `2.0.0-preview.99`)
+1. `git fetch origin main:main` to fetch the latest `main` branch
+1. `git checkout -b "release-$FEDERATION_RELEASE_VERSION" main`
+1. Update `CHANGELOG.md` in each repo ([more info](#changelogs))
+1. `git add ':/*CHANGELOG.md'` to add only changelogs to the commit
+1. `git commit --message "Update CHANGELOGs for v$FEDERATION_RELEASE_VERSION release"`
+1. `git push -u origin "release-$FEDERATION_RELEASE_VERSION"`
+1. `open "https://github.com/apollographql/federation/compare/main...release-$FEDERATION_RELEASE_VERSION?quick_pull=1&title=Release+$FEDERATION_RELEASE_VERSION&template=APOLLO_RELEASE_TEMPLATE.md"` to open PR in Github
+
+#### Bump version and tag release
+
+1. `npx lerna version --no-push --force-publish=\* "$FEDERATION_RELEASE_VERSION"` ([more info](#lerna-version))
```

Comment:
We've force published in the past in order to lockstep versioning of related packages but moved away from that awhile ago to reduce noise. I think there are good reasons for both but mostly just want to acknowledge this change. Is this something we'll continue to do after v2 becomes `latest`?

Reply:
I appreciate the extra context. I don't feel strongly about it.

I would say that it made it easy to think about and deploy if everything is at the same version. It felt kinda awkward to say we're getting ready for a `preview.3` release, even though some of the components were at some other version.

Though I think the noise aspect could get overwhelming downstream. Happy to take out the `force-publish` and see how that goes.

Reply:
To add to the lock-step version bumping camp, it seems like it was mostly already discussed https://apollographql.quip.com/BzkQAXFw2niK/045-Supergraph-Versioning-for-Fed-2-PreviewGA. The way I read that, is that we'll use the same version across components.

I also like the thought of potentially consolidating a few packages.